lib/models/node.js

```diff
@@ -64,6 +69,31 @@ function NodeModelFactory (Model, waterline, _) {
                     return !_.isEmpty(catalog);
                 });
             }
+        },
+        addTags: function(id, tags) {
+            return Promise.fromNode(this.native.bind(this))
+                .then(function(collection) {
+                    return collection.update(
+                        { _id: waterline.nodes.mongo.objectId(id) },
+                        { 
+                            $addToSet: { tags: { $each: tags } },
+                            $set: { updatedAt: new Date() }
+                        });
+                });
+        },
+        remTags: function(id, tag) {
```

Comment:
Yeah, I don't really have a problem with remTags, it's just like "delX" which you see in a lot of places.

Also @zyoung51 if you get the chance will you add a // TODO above this to refactor it into this.updateMongo once my workflow PR goes in?